src/components/CondoCardModal.vue

```diff
@@ -74,14 +74,33 @@ export default {
       this.$store.commit('setActiveFeature', null);
     },
     addCondoRecords(state) {
-      if(this.$store.state.lastSearchMethod === "geocode") {
+      let mapUnitIds = function(id) {
         var i = 0;
-        this.$store.state.condoUnits.units[this.$store.state.activeCondo.featureId].map(
-          a => {a._featureId = a.properties.pwd_parcel_id + "-UNIT-" + i, i++}
+        let unitsToAdd = this.$store.state.condoUnits.units[id]
+        unitsToAdd.map(
+          a => {
```

Comment:
Can we use a more descriptive variable name than `a`? This makes this method a bit hard to read.

Reply:
Sure. You'll probably see this a lot with simple mapping to properties. 
I tend to type it before I even think about it.